llmproxy/utils/category_model.py

```diff
@@ -0,0 +1,28 @@
+from transformers import pipeline
+
+class CategoryModel:
```

Comment:
Originally this was the class that would have called the trained model with the user prompt passed in. It would have decoded it the output as well but since that model will be used in the future, I decided to keep that class and put the temp implementation in for now. It'll make the inevitable future migrating a tad bit less annoying.